src/main/java/backend/Model.java

```diff
@@ -102,7 +107,7 @@ public LR_Model(Instances d, String[] params) throws Exception {
     protected void run() throws Exception {
         //invoke crossfold validation (Classifier obj, Instance, #folds, RNG)
         eval.crossValidateModel(classifier, data, 10, new Random(1));
```

Comment:
Not all of our classifiers will use this exact method (eval.crossValidateModel) so I've left our method abstract in the base Model.

Further, we may want to have different types of run() in the future, so cramming it into the abstract class or creating a middle-layer abstraction would just cause undue suffering.